src/musicbrainz.js

```diff
@@ -38,9 +35,96 @@ exports.track = (name, artist) => {
     }, (response) => {
       var body = "";
       response.on('data', (chunk) => body += chunk)
-      response.on('end', () => resolve(JSON.parse(body)))
+      response.on('end', () => resolve(JSON.parse(body)));
     })
   }).then((json) => {
-    return json.recordings[0]
+      let sortedRecordings = json.recordings.sort((a, b) => a.score - b.score);
+      return sortedRecordings[sortedRecordings.length-1];
   });
 }
+
+
+exports.coverArt = (recording) => {
+    let release = getBestRelease(recording)["release-group"]["id"];
+    if(release) {
+        return new Promise((resolve, reject) => {
+            http.get({
+                protocol: "http:",
+                hostname: "coverartarchive.org",
+                headers: {
+                    "User-Agent": "Application 88nine/0.1.0 (radiomilwaukee.org)",
+                    "Accept": "application/json"
+                },
+                path: `/release-group/${release}`,
+            }, (response) => {
+                if(response.statusCode == 200) {
+                    var body = "";
+                    response.on('data', (chunk) => body += chunk)
+                    response.on('end', () => resolve(JSON.parse(body)))
+                } else {
+                    resolve(null)
+                }
+            })
+        }).then((json) => {
+            return json
+        });
+    } else {
+        return new Promise((resolve, reject) => resolve({}))
+    }
+}
+
+
+function getBestRelease(recording) {
```

Comment:
Music Brains has the concept of "Release" and "Release Group"  an track can be on multiple different albums or "releases". So it returns shit like "Music Jam 2018" as an release. So we can't just use the first release always. 

A release group is like when an album is released multiple times but its really the same album, say for different countries or different era's. 

SO What this does is try a bunch of different methods based on idealness. My best case is a release group that is represented multiple times we use that cause its probably the best one.